paddle/fluid/operators/fused/fused_attention_op.cc

```diff
@@ -157,17 +197,32 @@ class FusedAttentionOp : public framework::OperatorWithKernel {
       ctx->SetOutputDim("Ln2Variance", {x_dim[0] * x_dim[1]});
       ctx->SetOutputDim("BiasDropoutResidualOut", ctx->GetInputDim("X"));
     }
-    // [batch_size, seq_len, 3, num_head, head_size]
-    ctx->SetOutputDim("QKVOut",
-                      {x_dim[0], x_dim[1], y_dim[0], y_dim[1], y_dim[2]});
 
-    if (ctx->HasInput("QKVBias")) {
-      ctx->SetOutputDim("QKVBiasOut",
+    if (!transpose_qkv_wb) {
```

Comment:
I prefer to unifying the following codes. Two many if-else branches.

Reply:
Ok, already updated